### PR TITLE
Add Guidelines on Documenting Comments

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add guidelines on documenting comments.

Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/1208

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

**Review by lafernando — APPROVED** [review]

File: 1.1/learn/how-to-document-ballerina-code.md [hunk 1]
```diff
@@ -64,6 +64,19 @@ The supported structure of documentation syntax is as follows:
 #            <return_parameter_description_line_2>
 ```
 
+>**Tip:** As a best practice, do not add a full stop at the end of the first sentence of the parameter and return type descriptions. Instead, add a fullstop only at the end of a function description. For Example,
```
Comment (anupama-pathirage) [inline 1.1/learn/how-to-document-ballerina-code.md:67]: IMHO, this description is bit misleading. It says `do not add a full stop at the end of the first sentence of the parameter and return type descriptions.` 

But actually we should add it to the first sentence if there are two sentences. We need to omit if only if we have only one sentence.

File: 1.1/learn/how-to-document-ballerina-code.md [hunk 2]
```diff
@@ -64,6 +64,19 @@ The supported structure of documentation syntax is as follows:
...
+
+```
+# Description for the function.
+#
+# + i - One sentence only
+# + s - Sentence one. Sentence two.
+# + return - Return description
+public function foo(int i, string s) returns boolean {
+return true;
```
Comment (anupama-pathirage) [inline 1.1/learn/how-to-document-ballerina-code.md:76]: The code is not properly formatted. Check all the places.

File: 1.1/learn/how-to-document-ballerina-code.md (hunk 1 above)
Comment (lafernando) [reply 1.1/learn/how-to-document-ballerina-code.md:67]: +1. @praneesha please add this.

File: 1.1/learn/how-to-document-ballerina-code.md (hunk 2 above)
Comment (lafernando) [reply 1.1/learn/how-to-document-ballerina-code.md:76]: @praneesha please load these sample code snippets to vscode, do a code format, and use it after that.

Comment (praneesha) [reply 1.1/learn/how-to-document-ballerina-code.md:76]: Done.

File: 1.1/learn/how-to-document-ballerina-code.md (hunk 1 above)
Comment (praneesha) [reply 1.1/learn/how-to-document-ballerina-code.md:67]: Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/1269.

File: 1.1/learn/how-to-document-ballerina-code.md (hunk 2 above)
Comment (praneesha) [reply 1.1/learn/how-to-document-ballerina-code.md:76]: Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/1269.

**Pull request merged**